plugins/out_splunk/splunk.c

```diff
@@ -230,6 +230,13 @@ static void cb_splunk_flush(const void *data, size_t bytes,
                             ctx->auth_header, flb_sds_len(ctx->auth_header));
     }
 
+    /* Append Channel identifier header */
+    if (ctx->channel) {
+        flb_http_add_header(c, FLB_SPLUNK_CHANNEL_IDENTIFIER_HEADER,
+                            strlen(FLB_SPLUNK_CHANNEL_IDENTIFIER_HEADER),
```

Comment:
here you can do ```sizeof(FLB_SPLUNK_CHANNEL_IDENTIFIER_HEADER) - 1``` since is not a variable length

Reply:
OK. Thank you.